### ISSUE TITLE
[FEATURE] Auto-Enable Setting on Search Bar

### ISSUE THREAD

**Issue by storm772** [John200410/rusherhack-issues#1109]
**What feature would you like to be added.**
Add a option called "Auto Enable" which when you hit Enter (after clicking in some module) the module will get activated.

**Is your feature request related to a problem? Please describe.**
Sometimes i just can't find the module that Search highlighted because im dumb

**Additional context**
Also, add a (Highlight color (which will just change the color of the highlight box) and a Highlight Notification, that will pop-up a message on screen saying like: "AutoCrystal module is located on Combat." )

**Comment by TwoThreeTwo** [comment]
When Search highlights the module you're looking for, it will show what category it is in, there's no need to make a notification that states where the module is, it also scrolls down and highlights the module, if you can not see that I suggest making your GUI bigger

**Comment by Luigit12** [comment]
Yes, sometimes i just dont want to move my mouse and click on the module

**Comment by storm772** [comment]
i want a option that auto enables the thing i have searched
